### ISSUE TITLE
No Notifications received on iOS

### ISSUE THREAD

**Issue by SenexTech** [thudugala/Plugin.LocalNotification#283]
Not getting any notifications on iOS device when I create a Notification using a blank MAUI  VS2022 template.   Notifications work on Android but not iOS.

**To Reproduce**
1. Create a blank MAUI project in VS2022
2. Install the Plugin.LocalNotification 9.2.1-preview05 using Nuget
3. Add  "LocalNotificationCenter.Setup();" in Appdelegate.cs
4. Add " await LocalNotificationCenter.ResetApplicationIconBadgeNumber(application);" in "WillEnterForeground" method
5. Create notification in codebehind
  var notification = new NotificationRequest
        {
            Title = "Notification Time: " + time.ToString(),
            Description = "Created: " + dt.ToShortDateString(),
            ReturningData = "Data",
            Subtitle = "Should trgigger at: " + dt.AddMinutes(Int16.Parse(time)).ToString(),
            NotificationId = id,
           
            iOS=
            {
                HideForegroundAlert = false,
                PlayForegroundSound = true
            },
            Schedule =
                        {
                            NotifyTime = dt.AddSeconds(5) // Used for Scheduling local notification
                        }
        };

6.  Create notification: await LocalNotificationCenter.Current.Show(notification); in code behind

**Expected behavior**
Notification should appear in 5 seconds

**Smartphone (please complete the following information):**
 - iPhone SE
 -  iOs 15.2.1 iPhone

**Comment by thudugala** [comment]
@SenexTech can you please attach a sample project?

**Comment by SenexTech** [comment]
[TestNotifications.zip](https://github.com/thudugala/Plugin.LocalNotification/files/8899801/TestNotifications.zip)
Here is a sample project

**Comment by thudugala** [comment]
@SenexTech 

Can you try this for me?

No Schedule, No iOS setting change 

```cs
var notification = new NotificationRequest
{
    Title = "Notification Time: " + time.ToString(),
    Description = "Created: " + dt.ToShortDateString(),
    ReturningData = "Data",
    Subtitle = "Should trgigger at: " + dt.AddMinutes(Int16.Parse(time)).ToString(),
    NotificationId = id
};

await LocalNotificationCenter.Current.Show(notification);
```

**Comment by SenexTech** [comment]
I beleieve I found the issue.  It looks like it was working but notifications  were being delayed an additional 4 hours from what I set.  It seems there is an issue with DateTime.Now in MAUI in that it is giving time in UTC (not local) which in my case is 4 hours ahead.  So my notifications were being delay by 4 hours.  I adjusted for the 4 hours and all works great.  I see an issue has aleady been posted in MAUI GitHub.  Thanks for your help.
